sklearn/manifold/tests/test_mds.py

```diff
@@ -55,10 +55,103 @@ def test_smacof_error():
         mds.smacof(sim, init=Z, n_init=1)
 
 
+def test_svd():
+    # Test svd using example data from "An Introduction to MDS"
+    # Florian Wickelmaier, p 11
+    sim = np.array([[0, 93, 82, 133],
+                    [93, 0, 52, 60],
+                    [82, 52, 0, 111],
+                    [133, 60, 111, 0]])
+
+    X, stress = mds.svd_scaler(sim, n_components=2)
+    X_true_1 = np.array([[-62.831, -32.97448],
+                         [18.403, 12.02697],
+                         [-24.960, 39.71091],
+                         [69.388, -18.76340]])
+    X_true_2 = np.copy(X_true_1)
+    X_true_2[:, 0] *= -1
+
+    # Signs of columns are dependent on signs of computed eigenvectors
+    # which are arbitrary and meaningless
+    assert(np.allclose(X, X_true_1)
```

Comment:
Let's just compare the absolute values then

Reply:
Is there any function that compares matrixes in respect to abs of their corresponding columns? Cause 'absing' two matrixes and comparing them is not what we want.

Reply:
I understand but it's extremely unlikely that `mds.svd_scaler` would come up with a wrong matrix that happens to still pass the absolute value check, right?

Reply:
Unless I'm drunk and set output[0, 8] *= -1. I think the point of testing is to avoid as many stupid mistakes as possible, while keeping costs affordable. Here we can protect against me being drunk with two lines. If You wish, I can change that, but I'd rather not.